src/Route.php

```diff
@@ -160,6 +160,12 @@ public function getOptions() : array
      */
     private function validateHttpMethods(array $methods) : array
     {
+        if (empty($methods)) {
```

Comment:
We can have here just `! $methods`

Reply:
Considering the argument is typed as an array, this makes it more semantically clear that we're checking to see if we have an _empty_ array.

Reply:
the result will be exactly the same and everywhere we are using just `!`, but fine - whatever you prefer, both works the same.

Reply:
I'm aware the results will be the same. Readability and intent are important, however!